mapclassify/value_by_alpha.py

```diff
@@ -0,0 +1,507 @@
+"""
+functionality originally written by Stefanie Lumnitz <stefanie.lumitz@gmail.com>
+as part of the ``splot`` package.
+
+TODO:
+* add Choropleth functionality with one input variable
+* merge all alpha keywords in one keyword dictionary
+for vba_choropleth
+"""
+
+import collections.abc
+
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import colormaps as cm
+from matplotlib import colors, patches
+
+from ._classify_API import classify
+
+__author__ = "Stefanie Lumnitz <stefanie.lumitz@gmail.com>"
+
+
+############################################################
+#
+# stuff from ``splot/_viz_value_by_alpha_mpl.py
+#
+############################################################
+
+
+def vba_choropleth(
```

Comment:
Docstring could benefit from an explanation of what vba does and when a user might use it.